src/som/interpreter/actors/Actor.java

```diff
@@ -122,13 +130,21 @@ protected void logNoTaskForActor() { }
   @TruffleBoundary
   public final synchronized void send(final EventualMessage msg) {
     assert msg.getTarget() == this;
-    mailbox.append(msg);
-    logMessageAddedToMailbox(msg);
+    if (msg.hasPause()) {
+      VM.println("Message pause--");
```

Comment:
Please do not commit debugging output.
In git it should be possible to select specific lines not to be included in a commit.